pymc3/step_methods/smc.py

```diff
@@ -28,7 +28,7 @@
 from .arraystep import metrop_select
 from ..backends import smc_text as atext
 
-__all__ = ['SMC', 'ATMIP_sample']
+__all__ = ['SMC', 'smc_sample']
```

Comment:
And according to pep8 acronyms in class names should only have the first letter capitalized: http://legacy.python.org/dev/peps/pep-0008/#descriptive-naming-styles
I guess that means that acronyms in function names should be all lower case (which is the case in the newer python stdlib modules I think, eg `ipaddress`).